web/src/clipboard_handler.ts

```diff
@@ -0,0 +1,45 @@
+import * as blueslip from "./blueslip";
+import {try_stream_topic_syntax_text} from "./copy_and_paste";
+
+export function clipboard_handler($link_element: JQuery, hide_popover: () => void): void {
+    if (typeof $link_element.data("clipboard-text") === "string") {
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+        const stream_topic_link_element_data = $link_element.data("clipboard-text") as string;
+        const stream_topic_syntax_text = try_stream_topic_syntax_text(
+            stream_topic_link_element_data,
+        );
+
+        // invalid links or stream topic names containing markdown cause this variable to be null
+        if (stream_topic_syntax_text === null) {
+            navigator.clipboard
+                .writeText(stream_topic_link_element_data)
+                .then(hide_popover)
+                .catch(() => {
+                    blueslip.error("Clipboard access denied: requires user interaction.");
+                });
+            return;
+        }
+        // The try_stream_topic_syntax_text() function returns the text
+        // as  #**stream_name > topic_name** to remove those two stars
+        // on fields supporting html in the start and end  we use
+        // replaceAll method that replaces those stars with html entity
+        // that is invisible however markdown still works on the zulip chat
```

Comment:
This comment block is misplaced

Reply:
Let's just remove this entirely.

Reply:
I used linter for it it is placed using built in linter And it was not there tim Abbot told me to leave a comment here so developer can know what's  going on here

Reply:
The line of code that comment talks about is few lines above the comment.
And in any case it is too verbose for what it is describing